docs/data/material/getting-started/templates/dashboard/components/CustomizedTreeView.tsx

```diff
@@ -118,7 +118,7 @@ function CustomLabel({ color, expandable, children, ...other }: CustomLabelProps
     >
       {iconColor && <DotIcon color={iconColor} />}
 
-      <Typography className="labelText" variant="body2">
```

Comment:
I added the `color` here because the text was dark even on dark mode—maybe there's a better solution, but in the meantime it solves for both custom theme and default theme